teemaderegister-react/src/components/AdminTos.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Input, Button, Typography, Form, Layout, Space } from 'antd'
+
+const { TextArea } = Input
+const { Title, Paragraph } = Typography
+
+const { Content } = Layout
+
+const AdminTos = props => {
+  const [editable, setEditable] = useState(false)
+  const [content, setContent] = useState(props.content)
+
+  useEffect(() => {
+    props.getTos()
+  }, [])
+
+  const flipEditable = () => setEditable(!editable)
+  const onFinish = values => {
+    setContent(values.textInput)
+    props.saveTos({ content: values.textInput })
```

Comment:
currently it does saving on background, maybe it would be good to disable edit button and form until saving is complete. But for this You can also keep it as is.